src/org/labkey/test/tests/TriggerScriptTest.java

```diff
@@ -802,13 +802,11 @@ private void setupDataClass()
             drt.clickHeaderButtonAndWait("Delete");
             clickButton("Confirm Delete");
         }
-        drt.clickInsertNewRow();
-        setFormElement(Locator.name("name"), DATA_CLASSES_NAME);
-        clickButton("Create");
-        DomainDesignerPage domainDesignerPage = new DomainDesignerPage(getDriver());
-        DomainFormPanel domainFormPanel = domainDesignerPage.fieldsPanel();
-        domainFormPanel.addField(COMMENTS_FIELD).setLabel(COMMENTS_FIELD);
-        domainFormPanel.addField(COUNTRY_FIELD).setLabel(COUNTRY_FIELD);
-        domainDesignerPage.clickFinish();
+        drt.clickHeaderMenu("New Data Class", "Design Manually");
+
+        CreateDataClassPage createDataClassPage = new CreateDataClassPage(getDriver());
+        createDataClassPage.setName(DATA_CLASSES_NAME)
+            .addFields(List.of(new FieldDefinition(COMMENTS_FIELD), new FieldDefinition(COUNTRY_FIELD)))
+            .clickSave();
```

Comment:
Consider creating this data class via API:
```suggestion

        DataClassDefinition dataClass = new DataClassDefinition(DATA_CLASSES_NAME)
                .setFields(List.of(new FieldDefinition(COMMENTS_FIELD), new FieldDefinition(COUNTRY_FIELD)));
        TestDataGenerator.createDomain(getProjectName(), dataClass);
```

Reply:
Yeah, I can do that. I think this might actually be one of the only places in the selenium tests that use the UI for data class creation, so we should definitely add a new set of test cases for that at some point. @labkey-tchad is there a test backlog place that makes sense to add that item (i.e. more selenium test coverage of the Data Class create/edit via UI)?

Reply:
Dan added some UI automation in SM but I aggree, we should have some LKS coverage as well. I'd say add a new test story to the existing epic (6961)